src/payment/instrument/instrument-action-creator.spec.ts

```diff
@@ -73,7 +77,23 @@ describe('InstrumentActionCreator', () => {
 
             expect(instrumentRequestSender.getVaultAccessToken).toHaveBeenCalled();
             expect(instrumentRequestSender.loadInstruments).toHaveBeenCalledWith(
-                { storeId, customerId, authToken: vaultAccessToken },
+                { storeId, customerId, currencyCode, authToken: vaultAccessToken },
+                shippingAddress
+            );
+        });
+
+        it('sends a request to get a list of instruments when shopper currency is not transactional', async () => {
```

Comment:
`sends a request to get a list of instruments using default currency when shopper currency is not transactional`